src/view/com/pager/PagerWithHeader.web.tsx

```diff
@@ -121,30 +121,19 @@ let PagerTabBar = ({
   onSelect?: (index: number) => void
   tabBarAnchor?: JSX.Element | null | undefined
 }): React.ReactNode => {
-  const pal = usePalette('default')
-  const {isMobile} = useWebMediaQueries()
   return (
     <>
-      <View
-        style={[
-          !isMobile && styles.headerContainerDesktop,
-          pal.border,
-          !isHeaderReady && styles.loadingHeader,
```

Comment:
I recall this had some purpose in the past, is that no longer true? Maybe it's only needed on native? It would be good to understand this.

Reply:
Chatted in slack 👍

Reply:
Added to doc here https://github.com/bluesky-social/social-app/pull/6907/commits/8914f1e60d01839577d9fe30a2b87304733761f5